docs/parts/wokwi-hx711.md

```diff
@@ -0,0 +1,97 @@
+---
+title: wokwi-hx711 Reference
+sidebar_label: wokwi-hx711
+---
+
+HX711 Load Cell Amplifier
+
+<wokwi-hx711 type="50kg" />
+
+## Pin names
+
+| Name | Description                                       |
+| ---- | ------------------------------------------------- |
+| VCC  | Voltage supply (5V)                               |
+| DT   | Serial data                                       |
+| SCK  | Serial clock                                      |
+| GND  | Ground                                            |
+
+*note that `E+`/`E-`/`A+`/`A-`/`B+`/`B-` pins are non-interactive and rendered based on attributes*  
+
+<wokwi-hx711 type="5kg" />
```

Comment:
We usually create a table at the bottom of the "Attributes" section with examples of how the attributes affect the visual style of the element, e.g. [here](https://docs.wokwi.com/parts/wokwi-7segment#examples)